packages/terra-worklist-data-grid/src/subcomponents/Row.jsx

```diff
@@ -159,7 +159,7 @@ function Row(props) {
 
   return (
     <tr
-      className={cx([isSelected ? 'row-selected' : 'worklist-data-grid-row', theme.className])}
+      className={cx([isSelected ? 'row-selected' : 'worklist-data-grid-row', hasRowSelection ? 'selectable' : null, theme.className])}
```

Comment:
You would want a pattern like this for your conditional class names

```
cx('worklist-data-grid-cell', {
    masked: isMasked,
    selectable: isSelectable && !isMasked,
    selected: isSelected && !isMasked,
    blank: !children,
  }, theme.className);
```

Reply:
Updated